examples/ktor/README.md

```diff
@@ -0,0 +1,32 @@
+# Containerize a [Ktor] application with Jib
+
+This is an example of how to easily build a Docker image for a [Ktor] application with Jib.
+
+```shell
+./gradlew jibDockerBuild
+
+docker run -d --rm -p 8080:8080 ktor-jib-example:1
```

Comment:
Let's remove `-d`. Will show logs in the terminal and the user can do Ctrl+C to quit.

Reply:
Whoops. Must've copied that line from me testing other things